vars/wazuh-manager.yml

```diff
@@ -0,0 +1,152 @@
+---
+# Wazuh-Manager, Filebeat and Open Distro role configuration
+# Reference: https://documentation.wazuh.com/4.1/deploying-with-ansible/reference.html
+# Defaults:
+# - https://github.com/wazuh/wazuh-ansible/blob/v4.1.5/roles/opendistro/opendistro-elasticsearch/defaults/main.yml
+# - https://github.com/wazuh/wazuh-ansible/blob/v4.1.5/roles/wazuh/ansible-wazuh-manager/defaults/main.yml
+# - https://github.com/wazuh/wazuh-ansible/blob/v4.1.5/roles/wazuh/ansible-filebeat-oss/defaults/main.yml
+# - https://github.com/wazuh/wazuh-ansible/blob/v4.1.5/roles/opendistro/opendistro-kibana/defaults/main.yml
+
+# Domain name used to generate Search Guard CE certificates
+domain_name: "wazuh.example.com"
+
+# Wazuh IP accessible from nodes running agents
+# Change network as needed
+wazuh_manager_ip: "{{ provision_oc_net_name | net_ip(inventory_hostname) }}"
+
+# Single node Open Distro setup
+single_node: true
+minimum_master_nodes: 1
+
+# All nodes are masters in a single node setup
+elasticsearch_node_master: true
+
+# Elasticsearch node.name
+# Also used in certificate file paths
+elasticsearch_node_name: "{{ inventory_hostname }}"
+
+# Elasticsearch network.host
+elasticsearch_network_host: "{{ wazuh_manager_ip }}"
+
+# Modify major versions at your own peril
+elastic_stack_version: "7.10.0"
```

Comment:
I got an error with a version mistach:

"Plugin wazuh [7.10.0] is incompatible with kibana version [7.10.2]". Possibly depends on version of role? Should we pin the version in requirements.yml? I noticed you did that for sms lab but requirements.yml is missing from this PR. Just wondered if this was a mistake or intentional.

Reply:
Mistake on my part! I'd updated smslab but forgot to port the fix. You're right that should be `elastic_stack_version: "7.10.2"`
The `requirements.yml` entry with the role was already included [here](https://github.com/stackhpc/kayobe-ops/pull/8).